pkgs/applications/misc/privacyidea/default.nix

```diff
@@ -52,7 +52,7 @@ let
         doCheck = false;
       });
       # Required by flask-1.1
-      jinja2 = super.jinja2.overridePythonAttrs (old: rec {
```

Comment:
why is this still needed?

Reply:
I don't remember path for this one, but jinja2 docs depended on it somewhere deep in dependency tree.